apps/nuxt3-ssr/public/_nuxt-styles/css/styles.umcg.css

```diff
@@ -772,10 +772,6 @@ video {
   margin-top: 1.25rem;
```

Comment:
this is a generated file and should be removed from git ( style using main.css and config files)